test/system/TODO.md

```diff
@@ -0,0 +1,112 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+# Overview
+
+System tests exercise podman in the context of a complete, composed environment from
+distribution packages.  It should match as closely as possible to how an end-user
+would experience a fresh-install.  Dependencies on external configuration and resources
+must be kept minimal, and the tests must be generic and vendor-neutral.
+
+The system-tests must execute cleanly on all tested platforms.  They may optionally
+be executed during continuous-integration testing of code-changes, after all other
+testing completes successfully.  For a list of tested platforms, please see [the
+CI configuration file.](../../.cirrus.yml)
+
+
+# Execution
+
+When working from a clone of [the libpod repository](https://github.com/containers/libpod),
+the main entry-point for humans and automation, is `make localsystem`.  When operating
+from a packaged version of the system-tests, the entry-point may vary and therefore
+must be documented for the platform.  Running the packaged system-tests assumes
+a matching version of podman, and all standard dependencies are installed.
+
+
+# Test Design and overview
+
+System-tests should be high-level and user work-flow oriented.  For example, consider
+how multiple podman invocations would be used together by an end-user.  This collection
+of related commands, and the end-result would be considered a single test.  If one or
+more of the intermediate commands fail, the test could still pass as long as the
+end-result is still achieved.
+
+# Matrix:
+
+- As-installed defaults, root user
+- As-installed defaults, regular user
+
```

Comment:
Maybe it will be clearer once you get the TODO's done, but I'm not sure what these two lines in the Matrix section stand for.

Reply:
Oh, we don't need this.  I'll just remove the whole section.